<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraDialog</class>
 <widget class="QDialog" name="CameraDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>카메라 설정</string>
  </property>
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    background-color: #f0f0f0;
    color: #333333;
}
QLabel {
    color: #333333;
}
QGroupBox {
    background-color: #f5f5f5;
    border-radius: 4px;
    border: 1px solid #c0c0c0;
    margin-top: 10px;
    color: #333333;
    font-weight: bold;
}
QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 5px;
}
QPushButton {
    background-color: #e0e0e0;
    color: #333333;
    border: 1px solid #c0c0c0;
    border-radius: 3px;
    padding: 6px;
    min-height: 25px;
}
QPushButton:hover {
    background-color: #d0d0d0;
}
QPushButton#connectButton {
    background-color: #4886e0;
    color: white;
    font-weight: bold;
}
QPushButton#disconnectButton {
    background-color: #e05a5a;
    color: white;
    font-weight: bold;
}
QComboBox, QSpinBox {
    background-color: #f5f5f5;
    color: #333333;
    border: 1px solid #c0c0c0;
    border-radius: 3px;
    padding: 3px;
    min-height: 25px;
}
QCheckBox {
    color: #333333;
}
QCheckBox::indicator {
    border: 1px solid #c0c0c0;
    border-radius: 2px;
    width: 14px;
    height: 14px;
}
QCheckBox::indicator:checked {
    background-color: #4886e0;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="cameraSettingsGroup">
     <property name="title">
      <string>카메라 설정</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="cameraSelectionLayout">
        <item>
         <widget class="QLabel" name="cameraLabel">
          <property name="text">
           <string>카메라:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cameraCombo"/>
        </item>
        <item>
         <widget class="QPushButton" name="refreshButton">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>새로고침</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="resolutionLayout">
        <item>
         <widget class="QLabel" name="resolutionLabel">
          <property name="text">
           <string>해상도:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="resolutionCombo"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="fpsLayout">
        <item>
         <widget class="QLabel" name="fpsLabel">
          <property name="text">
           <string>FPS:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="fpsSpinBox">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>60</number>
          </property>
          <property name="value">
           <number>30</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="saveSettingsCheckBox">
        <property name="text">
         <string>기본 설정으로 저장</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="connectButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>연결</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="disconnectButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>연결 해제</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>닫기</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>